packages/peregrine/lib/talons/CheckoutPage/useShippingMethod.js

```diff
@@ -171,6 +167,10 @@ export const useShippingMethod = props => {
         }
     }, [hasData, onSave]);
 
+    useEffect(() => {
+        setPageIsUpdating(isSettingShippingMethod);
```

Comment:
Shouldn't this be `isLoadingShippingMethods || isSettingShippingMethod`?

Reply:
No, this is a functionally identical change to what was happening before in the submit callback.